Libraries/LibWeb/HTML/Canvas/SerializeBitmap.cpp

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2025, Ladybird contributors
```

Comment:
We don't have "Ladybird contributors" as a standard copyright line. It's encouraged to use your own name + email, or "the Ladybird developers." if you're not comfortable with that for some reason.